### PR TITLE
Add information about excluding roles from reconciliation

### PR DESCRIPTION
**Pull request by liggitt**
Added in https://github.com/openshift/origin/pull/8774
post 1.2/3.2

### CODE REVIEW

**Comment by liggitt** [comment]
@adellape ping

File: install_config/upgrading/manual_upgrades.adoc [hunk 1]
```diff
@@ -147,6 +147,14 @@ your environment, you can run:
 # oadm policy reconcile-cluster-roles
 ----
 
+[WARN]
```
Comment (ahardin-rh) [inline install_config/upgrading/manual_upgrades.adoc]: s/WARN/WARNING

**Comment by ahardin-rh** [comment]
@liggitt Thanks! just one comment from me. Also, please check my label and milestone.  :bow:

**Comment by liggitt** [comment]
fixed, milestone looks good

**Comment by ahardin-rh** [comment]
[rev_history]
|link:../install_config/upgrading/manual_upgrades.html[Performing Manual Cluster Upgrades]
|Add a Warning box about excluding roles from reconciliation.
%

**Pull request merged**